Detections/ASimProcess/SdeletedeployedviaGPOandrunrecursively(ASIMVersion).yaml

```diff
@@ -56,14 +56,14 @@ entityMappings:
     fieldMappings:
       - identifier: Address
         columnName: DvcIpAddr
-version: 1.0.5
+version: 1.0.6
 kind: Scheduled
 metadata:
     source:
         kind: Community
     author:
-        name: Microsoft Security Research
+        name: Kosta Sotic, Microsoft
```

Comment:
Since this is an existing detection, please keep the same author

Reply:
Updated to keep the same author @vakohl